src/path_manager.c

```diff
@@ -1050,20 +1072,20 @@ struct mptcpd_pm *mptcpd_pm_create(struct mptcpd_config const *config)
         }
 
         if (l_genl_add_family_watch(pm->genl,
-                                       MPTCP_GENL_NAME,
-                                       family_appeared,
-                                       family_vanished,
-                                       pm,
-                                       NULL) == 0
+                                    name,
+                                    family_appeared,
+                                    family_vanished,
+                                    pm,
+                                    NULL) == 0
             || !l_genl_request_family(pm->genl,
-                                      MPTCP_GENL_NAME,
+                                      name,
                                       family_appeared,
                                       pm,
                                       NULL)) {
                 mptcpd_pm_destroy(pm);
-                l_error("Unable to watch or request \""
-                        MPTCP_GENL_NAME
-                        "\" generic netlink family.");
+                l_error("Unable to watch or request \"%s\" "
+                        "generic netlink family.",
+                        name);
                 return NULL;
         }
 
```

Comment:
Another `"mptcp"` just below (line 1093, modified file)

Reply:
Got it.  I'll change it to be consistent with the other instances.